ledfx/devices/artnet.py

```diff
@@ -93,15 +101,19 @@ def config_use(self, config):
         self.data_max = self.num_devices * self.pixels_per_device
 
         self.channel_count = (
-            self.pre_amble.size
-            + (self.pixels_per_device * 3)
-            + self.post_amble.size
-        ) * self.num_devices
+            self.dmx_start_address
+            + (
+                self.pre_amble.size
+                + (self.pixels_per_device * 3)
+                + self.post_amble.size
+            )
+            * self.num_devices
+        )
 
         self.packet_size = self._config["packet_size"]
-        self.universe_count = (
-            self.channel_count + self.packet_size - 1
-        ) // self.packet_size
+        self.universe_count = math.ceil(
+            (self.channel_count) / self.packet_size
+        )
```

Comment:
Why import math to achieve this, when the prior implementation is a relatively common pattern for the same with purely integer math?

Reply:
Because it is more explicit. Performance doesn't matter here and for `math.ceil(self.channel_count / self.packet_size)` it is directly clear what it does. 
It is not obvious for me that `(self.channel_count + self.packet_size - 1) // self.packet_size` does the same.

Reply:
Noting I ask to understand. I will resolve this comment, both approaches are valid